docs/admin/templates/managing-templates/devcontainers/add-devcontainer.md

```diff
@@ -0,0 +1,143 @@
+# Add a dev container template to Coder
+
+A Coder administrator adds a dev container-compatible template to Coder
+(Envbuilder). This allows the template to prompt for the developer for their dev
+container repository's URL as a
+[parameter](../../extending-templates/parameters.md) when they create their
+workspace. Envbuilder clones the repo and builds a container from the
+`devcontainer.json` specified in the repo.
+
+You can create template files through the Coder dashboard, CLI, or you can
+choose a template from the
+[Coder registry](https://registry.coder.com/templates?tag=devcontainer):
```

Comment:
Does this filtering work for you? For me, it doesn't.

Reply:
it doesn't 👎 

```suggestion
[Coder registry](https://registry.coder.com/templates):
```